src/main/java/net/minecraftforge/event/world/BlockEvent.java

```diff
@@ -166,25 +168,53 @@ public void setExpToDrop(int exp)
     }
 
     /**
-     * Called when a block is placed by a player.
+     * Called when a block is placed.
      *
      * If a Block Place event is cancelled, the block will not be placed.
      */
     @Cancelable
-    public static class PlaceEvent extends BlockEvent
+    public static class BasePlaceEvent extends BlockEvent
```

Comment:
"Base" is not very specific here, maybe use "Entity" ?
```suggestion
    public static class EntityPlaceEvent extends BlockEvent
```

Reply:
Change has been made.